tst/CTA.Rules.Test/CTA.Rules.Test.csproj

```diff
@@ -30,6 +31,12 @@
     <None Update="CTAFiles\action.namespace.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="CTAFiles\Configs\Web.config">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="CTAFiles\Configs\authWeb.config">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
```

Comment:
I suggest adding to a different folder to avoid later issues. CTA files should be used for rule files only.